po/pt_BR.po

```diff
@@ -110,23 +118,28 @@ msgstr "Adicionar"
 
 #: ../src/add_dialog.c ../src/interface.c
 msgid "Add files"
-msgstr "Adicionar arquivos"
+msgstr "Adicionar os arquivos"
 
 #: ../src/main.c
 msgid ""
 "Add the given files by asking the name of\n"
 "                                     the archive and quit"
-msgstr "Adiciona os arquivos perguntando o nome do pacote e sai"
+msgstr ""
+"Adicionar os respetivos arquivos, perguntar\n"
+"                                     o nome do pacote e sair"
 
 #: ../src/main.c
 msgid ""
 "Add to archive by asking which files and\n"
 "                                     quit"
-msgstr "Adiciona arquivos ao pacote perguntando seus nomes e sai"
+msgstr ""
+"Adicionar ao pacote perguntando quais são\n"
+"                                     os arquivos e sair"
 
 #: ../src/window.c
 msgid "Adding files to archive, please wait..."
-msgstr "Adicionando arquivos ao pacote, aguarde por favor..."
+msgstr "Adicionando os arquivos ao pacote. Por "
+"favor, aguarde a finalização do processo."
```

Comment:
The original text ends with three dots (ellipsis) to indicate that an action has not yet been completed. Why is this missing from the translation?